test/users.test.js

```diff
@@ -74,7 +74,7 @@ describe('Users', () => {
       })
       .set('Accept', 'Application/JSON')
       .end((error, res) => {
-        adminToken = `Bearer ${res.body.token}`;
+        adminToken = `${res.body.token}`;
```

Comment:
why did we remove Bearer?

Reply:
Let me work on this as well

Reply:
@Mnickii I am worked your request. Its now okay.